src/envoy/http/service_control/handler_impl.cc

```diff
@@ -323,6 +359,8 @@ void ServiceControlHandlerImpl::callReport(
   }
   info.response_size = stream_info_.bytesSent() + response_header_size;
 
+  info.response_code_detail = stream_info_.responseCodeDetails().value_or("");
```

Comment:
So what are the cases where it will be empty? Only on successful requests now?

Reply:
For successful one, it will be "via_upstream".